### ISSUE TITLE
comment lier les vignettes et assurer la coherence des liens en fonction de la langue  ? 

### ISSUE THREAD

**Issue by ChristianWia** [phgrosjean/rfrench#50]
comment lier les vignettes et assurer la coherence des liens ? déja en utilisant le meme nom que la vignette cible et ensuite en s'assurant quelle  existe bien et ceci dans chaque langue.

exemple: 
datatable-secondary-indices-and-auto-indexing-en.Rmd se termine par "We will discuss fast subsets using keys and secondary indices to joins in the next _vignette, “Joins and rolling joins”._"
donc je voudrais voir un lien vers la next, flute y en na pas. Bon alors je vais chercher "Joins and rolling joins.Rmd" ou équivalent  : ça n'existe meme pas !

et si on pense francais, "Nous aborderons les sous-ensembles rapides utilisant des clés et des indices secondaires pour les joints dans la prochaine vignette, “Joints et jointures roulantes”."  où est la prochaine vignette ? je ne sais pas  et je ne parle pas de “Joints et jointures roulantes.Rmd”

**Comment by phgrosjean** [comment]
Oui, une fois que l’on aura décidé des noms finaux en français, on pourra homogénéiser dans le texte.

PhG

.........................................<°}))><........
 ) ) ) ) )
( ( ( ( (    Prof. Philippe Grosjean
 ) ) ) ) )
( ( ( ( (    Numerical Ecology
 ) ) ) ) )   Mons University, Belgium
( ( ( ( (
..............................................................

**Comment by ChristianWia** [comment]
la question était 'comment' pas 'quand' ,  c'est à dire quelles sont les méthodes existantes (si elles le sont) pour faire cela, leur  syntaxe (indépendamment des valeurs qui bien sur seront définies quand on aura statué)....  mais pas grave. J'ai étendu et développé le sujet en ajoutant les _anchor_ et posté cela au niveau supérieur sous le numéro [issues/6373](https://github.com/Rdatatable/data.table/issues/6373).
